content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -150,9 +148,10 @@ kubeadm join --token abcdef.1234567890abcdef --discovery-token-unsafe-skip-ca-ve
    tradeoff in your environment.
 
 #### File or HTTPS-based discovery
+
 This provides an out-of-band way to establish a root of trust between the control-plane node
-and bootstrapping nodes.   Consider using this mode if you are building automated provisioning
-using kubeadm.
+and bootstrapping nodes. Consider using this mode if you are building automated provisioning
+using kubeadm. The format of the discovery file is a regular Kubernetes [kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) file.
```

Comment:
What about adding a note about the fact that the discovery file can contain user credentials or not;
in case there are not credentials, the TLS discovery token is used